ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/monitor-sessions.yml

```diff
@@ -0,0 +1,40 @@
+---
+monitor_sessions:
+  - name: myMonitoringSession1
+    sources:
+      - name: Ethernet5
+        direction: both
+        access_group:
+          type: ip
+          name: ipv4ACL
+          priority: 10
+        ip_access_group_priority: 10
+      - name: Ethernet0
+        access_group:
+          type: ipv6
+          name: ipv6ACL
+          priority: 100
+    destinations:
+      - Ethernet48
+    encapsulation_gre_metadata_tx: true
+    header_remove_size: 32
+    truncate:
+      enabled: true
+  - name: myMonitoringSession2
+    sources:
+      - name: Ethernet12
+        direction: rx
+      - name: Ethernet18
+        access_group: 
```

Comment:
Remove the trailing space.

Please run the following command to ```pre-commit run -a``` to view all lint errors before committing any changes to github.

Reply:
Thanks Tony, very useful. Adding the check to my routine!